packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -341,7 +396,7 @@ abstract contract OPContractsManagerBase {
     }
 
     /// @notice Retrieves the AnchorStateRegistry address for a given SystemConfig
-    function getAnchorStateRegistry(ISystemConfig _systemConfig) internal view returns (IAnchorStateRegistry) {
+    function getAnchorStateRegistryV1(ISystemConfig _systemConfig) internal view returns (IAnchorStateRegistry) {
```

Comment:
This isn't actually V1 specific and is actually the best way to get the anchor state registry since it just goes to the portal rather than depending on a particular dispute game implementation.